test/lib/containerManager.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+var path = require('path');
+var request = require('request');
+var should = require('should');
+var level = require('level');
+
+var Loader = require('yaml-config-loader');
+var CM = require('../../lib/ContainerManager');
+var server = new CM();
+var loader = new Loader();
+var db = level('/tmp/db', {db: require('memdown')});
+
+const port = 9876;
+const reapedBuildId = 'a38c39ed-3bb9-44ba-9050-8a4337384d3b';
+const pendingBuildId = 'c65c469a-60a9-43bb-848c-e96242e6b610';
+const missingBuildId = 'fedcda29-cf3b-4ff3-83ce-308a632d477b';
+
```

Comment:
Yeah, I just noticed that as well. That was a mistake on my part. I like the idea of having all the build ids as constants at the top since they actually have to match with the ids in the nock file. I found this made it easier to keep track of everything. I'm open to moving them each into the test function associated with them it you feel strongly about it but this seemed cleaner to me and I thought that in further later tests it might make sense to use them again.